contracts/minitoken/solidity/package.json

```diff
@@ -18,6 +18,6 @@
     "truffle": "5.4.16"
   },
   "dependencies": {
-    "@hyperledger-labs/yui-ibc-solidity": "git+https://git@github.com/hyperledger-labs/yui-ibc-solidity.git#482544f9ca215f3a239fb814803d5819c9f1602c"
+    "@hyperledger-labs/yui-ibc-solidity": "git+https://git@github.com/hyperledger-labs/yui-ibc-solidity.git#d9d0f68e11f4b943706d92ab6861ccd432e2dac6"
```

Comment:
You should update this dependency.

Reply:
https://github.com/hyperledger-labs/yui-docs/pull/67/commits/ee63fc6dc095804b5d85d9ed3f20753509dba5e3
Chant to v0.3.1